docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+
+To begin, create a project directory and navigate into it:
+
+.. code-block:: console
+
+    $ mkdir time-series-go
+    $ cd time-series-go
+
+Next, choose a module path and create a `go.mod` file that declares it:
+
+.. code-block:: console
+
+    $ go mod init example.com/time-series-go
+
+You should see a `go.mod` file in the current directory with contents similar to:
+
+.. code-block:: console
+
+    module example.com/time-series-go
+
+    go 1.14
+
+Next, create a file named `main.go` in the same directory:
+
+.. code-block:: console
+
+   $ touch main.go
+
+Open up your favorite code editor. We will be editing this file a lot!
+
+
+Get ISS telemetry data
+======================
+
+You can get telemetry data from `Open Notify`_, a third-party service that
+provides a simple API to consume data from NASA (specifically, the current
+location of the International Space Station). The endpoint for this data is
+`<http://api.open-notify.org/iss-now.json>`_.
+
+
+In your `main.go` file, declare the main package at the top (to tell the 
+compiler that our program is an executable), import some packages from the 
+`standard library`_ that will be used in this tutorial and declare a main 
+function which will be the entry point of our executable program:
+
+.. code-block:: js
+
+    package main
+
+    import (
+        "encoding/json"
+        "fmt"
+        "io/ioutil"
+        "log"
+        "net/http"
+    )
+
+    func main() {
+
+    }
+
+Then, look at the JSON data that gets returned from going to the Open
+Notify API endpoint at `<http://api.open-notify.org/iss-now.json>`_. The 
+endpoint returns a JSON payload, which contains an ``iss_position`` object
+with ``latitude`` and ``longitude`` data.
+
+.. code-block:: js
+
+   {
+        "message":"success",
+        "timestamp":1591703638,
+        "iss_position":{
+            "longitude":"84.9504",
+            "latitude":"41.6582"
+        }
+    }
+
+The longitude and latitude of the International Space Station changes 
+constantly and is what you want to extract from this payload and insert into
+CrateDB. To parse this JSON, you can create a `struct`_ that can be used to 
+`unmarshal`_ the data into. When you unmarshal JSON into a struct, the 
+function matches incoming object keys to the keys in the struct field name
+or its tag. By default, object keys which don't have a corresponding struct 
+field are ignored.
+
+.. code-block:: js
+
+    type issInfo struct {
+        IssPosition struct {
+            Longitude string `json:"longitude"`
+            Latitude  string `json:"latitude"`
+        } `json:"iss_position"`
+    }
+
+Now, create a function that makes an HTTP GET request to the Open Notify API 
+endpoint and returns longitude and latitude as a `geo_point`_ declaration.
+
+The Go standard library comes with great support for HTTP client and server
+implementations in the `net/http`_ package. In this example you’ll use it to 
```

Comment:
"In this example you’ll use it to..." is a little confusing from a grammar pov, imo. it sounds like you are making a prediction about the future wrt the reader's actions

I would stick to constructions like this:

- "Issue an HTTP GET request to the API endpoint" (when giving an instruction)
- "You can use this package to issue an HTTP GET request to the API endpoint" (when informing the reader about what they can do)

this is something I have been struggling with on the tutorial I am currently working on. how to use second person while also not making it sound like I'm making predictions about what the reader will do. I have found that "You can use X for Y" is a good solution -- that way, the tutorial takes the form of a series of instructions and pointers about what the reader can do with the tools/technology being demonstrated

does this make sense? 😅